atlasdb-persistent-lock-api/src/main/java/com/palantir/atlasdb/sweeperservice/SweeperService.java

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Seems a little unusual to me that this lives in `persistent-lock-api`. 

Let's leave it here for now though. We should consider having a Sweep project, though that's out of the scope of this change!